lib/matplotlib/gridspec.py

```diff
@@ -30,9 +30,18 @@ class GridSpecBase:
 
     def __init__(self, nrows, ncols, height_ratios=None, width_ratios=None):
         """
-        The number of rows and number of columns of the grid need to
-        be set. Optionally, the ratio of heights and widths of rows and
-        columns can be specified.
+        Parameters
+        ----------
+        nrows, ncols : int
+            The number of rows and columns of the grid.
+        width_ratios : array-like of length *ncols*, optional
+            Defines the relative widths of the columns. Each column gets a
```

Comment:
As below: Remove "Defines" (throughought, actually)

Reply:
Since `width_ratios` accepts non-normalized values, I was being a bit more explicit. Strictly speaking `5, 2, 3` are not width ratios, but define the width ratios `0.5, 0.2, 0.3`.

Reply:
5:2:3 is definitely a ratio, see e.g. https://en.wikipedia.org/wiki/Ratio which starts with the example "For example, if a bowl of fruit contains eight oranges and six lemons, then the ratio of oranges to lemons is eight to six (that is, 8:6, which is equivalent to the ratio 4:3)."

Reply:
While @anntzer is technically correct, what @timhoffm wrote is clearer and requires less thought (for me) to understand.